lab_1/main.py

```diff
@@ -69,7 +96,36 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance = []
+    left_context = []
+    right_context = []
+    if isinstance(tokens, list) and isinstance(word, str) and type(left_context_size) == int\
```

Comment:
Usually, there are checks for arguments and body function - separated and on the same level.
Try to get rid of nesting.